### PR TITLE
Add descriptions to priority queue

### PR DESCRIPTION
**Pull request by heifner**
## Change Description

- Add description to the priority queue for debugging and profiling.
- The overhead when the description is not used should be minimal.
- Includes change to txn_text_gen_plugin to create two new accounts and to log private keys.
- Depends on https://github.com/EOSIO/appbase/pull/50

## Consensus Changes

None

## API Changes

None

## Documentation Additions

None

### CODE REVIEW

**Comment by spoonincode** [comment]
>Includes change to txn_text_gen_plugin to create two new accounts and to log private keys.

Why?

**Comment by heifner** [comment]
> Why?

As just a simple way to have two accounts with the same key. Log the keys so it is easy to grab the keys for use by other manual tests.

File: plugins/http_plugin/http_plugin.cpp [hunk 1]
```diff
@@ -300,7 +300,7 @@ namespace eosio {
                if( handler_itr != url_handlers.end()) {
                   con->defer_http_response();
                   bytes_in_flight += body.size();
-                  app().post( appbase::priority::low,
+                  app().post( appbase::priority::low, resource,
                               [ioc = this->server_ioc, &bytes_in_flight = this->bytes_in_flight, handler_itr,
                                resource{std::move( resource )}, body{std::move( body )}, con]() {
```
Comment (heifner) [inline plugins/http_plugin/http_plugin.cpp]: `resource` is moved into lambda. Need to copy so `resource` is available for description.

**Comment by heifner** [comment]
Going to close this for now. It was more for testing/debugging purposes.

**Pull request closed without merging**